src/satellite/client.ts

```diff
@@ -632,18 +635,20 @@ export class SatelliteClient extends EventEmitter implements Client {
           commit_timestamp: op.begin.commitTimestamp,
           lsn: op.begin.lsn,
           changes: [],
+          origin: op.begin.origin!,
```

Comment:
Good question. So on a direction from Satellite, origin is the same as an id which is passed in the Auth message. So it might very well be ignored by the server. However on the way back it is important to specify that, so that Satellite knows, when it receives transaction which originated from it.

Reply:
The comment in proto file explicitly says about it:
```
   // Globally unique id of the source that transaction originated from. For
    // data coming from Satellite this field is ignored. For data coming from
    // Electric this field can be used to deduce if the incoming transaction
    // originated on this Satellite instance or not.
```
Let me know if further clarification on the protocol is desired